lib/server_whitebox_test.go

```diff
@@ -29,12 +29,12 @@ const (
 	affiliationName = "org1"
 )
 
-// TestGetAffliation checks if there is one record for the
+// TestServerAffiliation checks if there is one record for the
 // affilition 'org1' in the database after starting the server
 // two times. This test is to make sure server does not create
 // duplicate affiliations in the database every time it is
 // started.
-func TestGetAffliation(t *testing.T) {
+func TestServerAffilication(t *testing.T) {
```

Comment:
If we're renaming, can we include the full method name?

`TestServerGetAllAffiliations` ?

Reply:
Thank you for your review. I got it. I will do that.